src/pages/api/answer.ts

```diff
@@ -1,23 +1,44 @@
 import {NextApiRequest, NextApiResponse} from 'next'
 import {convertkitAxios} from 'utils/axios-convertkit-api'
 import fetchConvertkitSubscriberFromServerCookie from 'utils/fetch-convertkit-subscriber'
+import isEmpty from 'lodash/isEmpty'
+import find from 'lodash/find'
 
 const answer = async (req: NextApiRequest, res: NextApiResponse) => {
   if (req.method === 'POST') {
     try {
-      console.log(req.body)
-      const {tagId} = req.body
+      const {tagId, survey} = req.body
       const cookieHeader = req.headers.cookie as string
-      const [
-        subscriber,
-        ckCookie,
-      ] = await fetchConvertkitSubscriberFromServerCookie(cookieHeader)
+      const siteNameShort = process.env.NEXT_PUBLIC_SITE_NAME_SHORT
```

Comment:
since these aren't in react and only used in API layer we can name the env vars `SITE_NAME_SHORT` without the NEXT_PUBLIC, unless that's a critical aspect.

I feel like these should probably be extracted into a "service" that accepts these params and would be testable. This would be true for the ck stuff below as well.

Personally I like to see these config strings "hoisted" to the top of the file with fallbacks and also errors thrown when they aren't present or have unexpected values.